crates/astria-sequencer-relayer/src/relayer.rs

```diff
@@ -164,6 +168,7 @@ impl Relayer {
         // Then report update the internal state or report if submission failed
         match submission_result {
             Ok(height) => self.state_tx.send_modify(|state| {
+                counter!("successful_submission_celestia_height", height);
```

Comment:
what does this do if metrics are not enabled? nothing?

Reply:
Yeah basically, without an exporter there's nowhere for them to go. We can also do this in libraries to implement metrics in them, if imported and exporter attached get the metrics if not nothing.